### ISSUE TITLE
Upgrade via pip to 1.0.1 failed on Ubuntu 17.10

### ISSUE THREAD

**Issue by Bu11etmagnet** [anaynayak/buildnotify#39]
```
$ python -V
Python 2.7.14

$ sudo -H pip install buildnotify --upgrade --no-clean
[sudo] password: 
Collecting buildnotify
  Downloading BuildNotify-1.0.1.tar.gz
Building wheels for collected packages: buildnotify
  Running setup.py bdist_wheel for buildnotify ... error
  Complete output from command /usr/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-9KxW3h/buildnotify/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/tmpZ94Amppip-wheel- --python-tag cp27:
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-2.7
  creating build/lib.linux-x86_64-2.7/buildnotifylib
  copying buildnotifylib/project_status_notification.py -> build/lib.linux-x86_64-2.7/buildnotifylib
  copying buildnotifylib/app_menu.py -> build/lib.linux-x86_64-2.7/buildnotifylib
  copying buildnotifylib/build_icons.py -> build/lib.linux-x86_64-2.7/buildnotifylib
  copying buildnotifylib/preferences.py -> build/lib.linux-x86_64-2.7/buildnotifylib
  copying buildnotifylib/server_configuration_dialog.py -> build/lib.linux-x86_64-2.7/buildnotifylib
  copying buildnotifylib/serverconfig.py -> build/lib.linux-x86_64-2.7/buildnotifylib
  copying buildnotifylib/config.py -> build/lib.linux-x86_64-2.7/buildnotifylib
  copying buildnotifylib/notifications.py -> build/lib.linux-x86_64-2.7/buildnotifylib
  copying buildnotifylib/app_ui.py -> build/lib.linux-x86_64-2.7/buildnotifylib
  copying buildnotifylib/version.py -> build/lib.linux-x86_64-2.7/buildnotifylib
  copying buildnotifylib/buildnotify.py -> build/lib.linux-x86_64-2.7/buildnotifylib
  copying buildnotifylib/app_notification.py -> build/lib.linux-x86_64-2.7/buildnotifylib
  copying buildnotifylib/__init__.py -> build/lib.linux-x86_64-2.7/buildnotifylib
  creating build/lib.linux-x86_64-2.7/buildnotifylib/core
  copying buildnotifylib/core/project.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
  copying buildnotifylib/core/continous_integration_server.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
  copying buildnotifylib/core/timed_event.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
  copying buildnotifylib/core/repeat_timed_event.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
  copying buildnotifylib/core/keystore.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
  copying buildnotifylib/core/filtered_continuous_integration_server.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
  copying buildnotifylib/core/background_event.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
  copying buildnotifylib/core/http_connection.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
  copying buildnotifylib/core/distance_of_time.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
  copying buildnotifylib/core/projects.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
  copying buildnotifylib/core/response.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
  copying buildnotifylib/core/__init__.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
  creating build/lib.linux-x86_64-2.7/buildnotifylib/generated
  copying buildnotifylib/generated/preferences_ui.py -> build/lib.linux-x86_64-2.7/buildnotifylib/generated
  copying buildnotifylib/generated/icons_rc.py -> build/lib.linux-x86_64-2.7/buildnotifylib/generated
  copying buildnotifylib/generated/__init__.py -> build/lib.linux-x86_64-2.7/buildnotifylib/generated
  copying buildnotifylib/generated/server_configuration_ui.py -> build/lib.linux-x86_64-2.7/buildnotifylib/generated
  running build_scripts
  creating build/scripts-2.7
  copying and adjusting buildnotifyapplet.py -> build/scripts-2.7
  changing mode of build/scripts-2.7/buildnotifyapplet.py from 644 to 755
  installing to build/bdist.linux-x86_64/wheel
  running install
  running install_lib
  creating build/bdist.linux-x86_64
  creating build/bdist.linux-x86_64/wheel
  creating build/bdist.linux-x86_64/wheel/buildnotifylib
  copying build/lib.linux-x86_64-2.7/buildnotifylib/project_status_notification.py -> build/bdist.linux-x86_64/wheel/buildnotifylib
  creating build/bdist.linux-x86_64/wheel/buildnotifylib/core
  copying build/lib.linux-x86_64-2.7/buildnotifylib/core/project.py -> build/bdist.linux-x86_64/wheel/buildnotifylib/core
  copying build/lib.linux-x86_64-2.7/buildnotifylib/core/continous_integration_server.py -> build/bdist.linux-x86_64/wheel/buildnotifylib/core
  copying build/lib.linux-x86_64-2.7/buildnotifylib/core/timed_event.py -> build/bdist.linux-x86_64/wheel/buildnotifylib/core
  copying build/lib.linux-x86_64-2.7/buildnotifylib/core/repeat_timed_event.py -> build/bdist.linux-x86_64/wheel/buildnotifylib/core
  copying build/lib.linux-x86_64-2.7/buildnotifylib/core/keystore.py -> build/bdist.linux-x86_64/wheel/buildnotifylib/core
  copying build/lib.linux-x86_64-2.7/buildnotifylib/core/filtered_continuous_integration_server.py -> build/bdist.linux-x86_64/wheel/buildnotifylib/core
  copying build/lib.linux-x86_64-2.7/buildnotifylib/core/background_event.py -> build/bdist.linux-x86_64/wheel/buildnotifylib/core
  copying build/lib.linux-x86_64-2.7/buildnotifylib/core/http_connection.py -> build/bdist.linux-x86_64/wheel/buildnotifylib/core
  copying build/lib.linux-x86_64-2.7/buildnotifylib/core/distance_of_time.py -> build/bdist.linux-x86_64/wheel/buildnotifylib/core
  copying build/lib.linux-x86_64-2.7/buildnotifylib/core/projects.py -> build/bdist.linux-x86_64/wheel/buildnotifylib/core
  copying build/lib.linux-x86_64-2.7/buildnotifylib/core/response.py -> build/bdist.linux-x86_64/wheel/buildnotifylib/core
  copying build/lib.linux-x86_64-2.7/buildnotifylib/core/__init__.py -> build/bdist.linux-x86_64/wheel/buildnotifylib/core
  copying build/lib.linux-x86_64-2.7/buildnotifylib/app_menu.py -> build/bdist.linux-x86_64/wheel/buildnotifylib
  creating build/bdist.linux-x86_64/wheel/buildnotifylib/generated
  copying build/lib.linux-x86_64-2.7/buildnotifylib/generated/preferences_ui.py -> build/bdist.linux-x86_64/wheel/buildnotifylib/generated
  copying build/lib.linux-x86_64-2.7/buildnotifylib/generated/icons_rc.py -> build/bdist.linux-x86_64/wheel/buildnotifylib/generated
  copying build/lib.linux-x86_64-2.7/buildnotifylib/generated/__init__.py -> build/bdist.linux-x86_64/wheel/buildnotifylib/generated
  copying build/lib.linux-x86_64-2.7/buildnotifylib/generated/server_configuration_ui.py -> build/bdist.linux-x86_64/wheel/buildnotifylib/generated
  copying build/lib.linux-x86_64-2.7/buildnotifylib/build_icons.py -> build/bdist.linux-x86_64/wheel/buildnotifylib
  copying build/lib.linux-x86_64-2.7/buildnotifylib/preferences.py -> build/bdist.linux-x86_64/wheel/buildnotifylib
  copying build/lib.linux-x86_64-2.7/buildnotifylib/server_configuration_dialog.py -> build/bdist.linux-x86_64/wheel/buildnotifylib
  copying build/lib.linux-x86_64-2.7/buildnotifylib/serverconfig.py -> build/bdist.linux-x86_64/wheel/buildnotifylib
  copying build/lib.linux-x86_64-2.7/buildnotifylib/config.py -> build/bdist.linux-x86_64/wheel/buildnotifylib
  copying build/lib.linux-x86_64-2.7/buildnotifylib/notifications.py -> build/bdist.linux-x86_64/wheel/buildnotifylib
  copying build/lib.linux-x86_64-2.7/buildnotifylib/app_ui.py -> build/bdist.linux-x86_64/wheel/buildnotifylib
  copying build/lib.linux-x86_64-2.7/buildnotifylib/version.py -> build/bdist.linux-x86_64/wheel/buildnotifylib
  copying build/lib.linux-x86_64-2.7/buildnotifylib/buildnotify.py -> build/bdist.linux-x86_64/wheel/buildnotifylib
  copying build/lib.linux-x86_64-2.7/buildnotifylib/app_notification.py -> build/bdist.linux-x86_64/wheel/buildnotifylib
  copying build/lib.linux-x86_64-2.7/buildnotifylib/__init__.py -> build/bdist.linux-x86_64/wheel/buildnotifylib
  running install_data
  creating build/bdist.linux-x86_64/wheel/BuildNotify-1.0.1.data
  creating build/bdist.linux-x86_64/wheel/BuildNotify-1.0.1.data/data
  creating build/bdist.linux-x86_64/wheel/BuildNotify-1.0.1.data/data/share
  creating build/bdist.linux-x86_64/wheel/BuildNotify-1.0.1.data/data/share/applications
  error: can't copy 'buildnotify.desktop': doesn't exist or not a regular file
  
  ----------------------------------------
  Failed building wheel for buildnotify
  Running setup.py clean for buildnotify
Failed to build buildnotify
Installing collected packages: buildnotify
  Found existing installation: BuildNotify 1.0.0
    Uninstalling BuildNotify-1.0.0:
      Successfully uninstalled BuildNotify-1.0.0
  Running setup.py install for buildnotify ... error
    Complete output from command /usr/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-9KxW3h/buildnotify/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-ypEgr8-record/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-2.7
    creating build/lib.linux-x86_64-2.7/buildnotifylib
    copying buildnotifylib/project_status_notification.py -> build/lib.linux-x86_64-2.7/buildnotifylib
    copying buildnotifylib/app_menu.py -> build/lib.linux-x86_64-2.7/buildnotifylib
    copying buildnotifylib/build_icons.py -> build/lib.linux-x86_64-2.7/buildnotifylib
    copying buildnotifylib/preferences.py -> build/lib.linux-x86_64-2.7/buildnotifylib
    copying buildnotifylib/server_configuration_dialog.py -> build/lib.linux-x86_64-2.7/buildnotifylib
    copying buildnotifylib/serverconfig.py -> build/lib.linux-x86_64-2.7/buildnotifylib
    copying buildnotifylib/config.py -> build/lib.linux-x86_64-2.7/buildnotifylib
    copying buildnotifylib/notifications.py -> build/lib.linux-x86_64-2.7/buildnotifylib
    copying buildnotifylib/app_ui.py -> build/lib.linux-x86_64-2.7/buildnotifylib
    copying buildnotifylib/version.py -> build/lib.linux-x86_64-2.7/buildnotifylib
    copying buildnotifylib/buildnotify.py -> build/lib.linux-x86_64-2.7/buildnotifylib
    copying buildnotifylib/app_notification.py -> build/lib.linux-x86_64-2.7/buildnotifylib
    copying buildnotifylib/__init__.py -> build/lib.linux-x86_64-2.7/buildnotifylib
    creating build/lib.linux-x86_64-2.7/buildnotifylib/core
    copying buildnotifylib/core/project.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
    copying buildnotifylib/core/continous_integration_server.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
    copying buildnotifylib/core/timed_event.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
    copying buildnotifylib/core/repeat_timed_event.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
    copying buildnotifylib/core/keystore.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
    copying buildnotifylib/core/filtered_continuous_integration_server.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
    copying buildnotifylib/core/background_event.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
    copying buildnotifylib/core/http_connection.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
    copying buildnotifylib/core/distance_of_time.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
    copying buildnotifylib/core/projects.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
    copying buildnotifylib/core/response.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
    copying buildnotifylib/core/__init__.py -> build/lib.linux-x86_64-2.7/buildnotifylib/core
    creating build/lib.linux-x86_64-2.7/buildnotifylib/generated
    copying buildnotifylib/generated/preferences_ui.py -> build/lib.linux-x86_64-2.7/buildnotifylib/generated
    copying buildnotifylib/generated/icons_rc.py -> build/lib.linux-x86_64-2.7/buildnotifylib/generated
    copying buildnotifylib/generated/__init__.py -> build/lib.linux-x86_64-2.7/buildnotifylib/generated
    copying buildnotifylib/generated/server_configuration_ui.py -> build/lib.linux-x86_64-2.7/buildnotifylib/generated
    running build_scripts
    creating build/scripts-2.7
    copying and adjusting buildnotifyapplet.py -> build/scripts-2.7
    changing mode of build/scripts-2.7/buildnotifyapplet.py from 644 to 755
    running install_lib
    copying build/lib.linux-x86_64-2.7/buildnotifylib/project_status_notification.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib
    copying build/lib.linux-x86_64-2.7/buildnotifylib/core/project.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib/core
    copying build/lib.linux-x86_64-2.7/buildnotifylib/core/continous_integration_server.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib/core
    copying build/lib.linux-x86_64-2.7/buildnotifylib/core/timed_event.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib/core
    copying build/lib.linux-x86_64-2.7/buildnotifylib/core/repeat_timed_event.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib/core
    copying build/lib.linux-x86_64-2.7/buildnotifylib/core/keystore.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib/core
    copying build/lib.linux-x86_64-2.7/buildnotifylib/core/filtered_continuous_integration_server.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib/core
    copying build/lib.linux-x86_64-2.7/buildnotifylib/core/background_event.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib/core
    copying build/lib.linux-x86_64-2.7/buildnotifylib/core/http_connection.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib/core
    copying build/lib.linux-x86_64-2.7/buildnotifylib/core/distance_of_time.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib/core
    copying build/lib.linux-x86_64-2.7/buildnotifylib/core/projects.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib/core
    copying build/lib.linux-x86_64-2.7/buildnotifylib/core/response.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib/core
    copying build/lib.linux-x86_64-2.7/buildnotifylib/core/__init__.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib/core
    copying build/lib.linux-x86_64-2.7/buildnotifylib/app_menu.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib
    creating /usr/local/lib/python2.7/dist-packages/buildnotifylib/generated
    copying build/lib.linux-x86_64-2.7/buildnotifylib/generated/preferences_ui.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib/generated
    copying build/lib.linux-x86_64-2.7/buildnotifylib/generated/icons_rc.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib/generated
    copying build/lib.linux-x86_64-2.7/buildnotifylib/generated/__init__.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib/generated
    copying build/lib.linux-x86_64-2.7/buildnotifylib/generated/server_configuration_ui.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib/generated
    copying build/lib.linux-x86_64-2.7/buildnotifylib/build_icons.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib
    copying build/lib.linux-x86_64-2.7/buildnotifylib/preferences.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib
    copying build/lib.linux-x86_64-2.7/buildnotifylib/server_configuration_dialog.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib
    copying build/lib.linux-x86_64-2.7/buildnotifylib/serverconfig.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib
    copying build/lib.linux-x86_64-2.7/buildnotifylib/config.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib
    copying build/lib.linux-x86_64-2.7/buildnotifylib/notifications.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib
    copying build/lib.linux-x86_64-2.7/buildnotifylib/app_ui.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib
    copying build/lib.linux-x86_64-2.7/buildnotifylib/version.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib
    copying build/lib.linux-x86_64-2.7/buildnotifylib/buildnotify.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib
    copying build/lib.linux-x86_64-2.7/buildnotifylib/app_notification.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib
    copying build/lib.linux-x86_64-2.7/buildnotifylib/__init__.py -> /usr/local/lib/python2.7/dist-packages/buildnotifylib
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/project_status_notification.py to project_status_notification.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/core/project.py to project.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/core/continous_integration_server.py to continous_integration_server.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/core/timed_event.py to timed_event.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/core/repeat_timed_event.py to repeat_timed_event.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/core/keystore.py to keystore.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/core/filtered_continuous_integration_server.py to filtered_continuous_integration_server.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/core/background_event.py to background_event.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/core/http_connection.py to http_connection.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/core/distance_of_time.py to distance_of_time.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/core/projects.py to projects.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/core/response.py to response.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/core/__init__.py to __init__.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/app_menu.py to app_menu.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/generated/preferences_ui.py to preferences_ui.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/generated/icons_rc.py to icons_rc.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/generated/__init__.py to __init__.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/generated/server_configuration_ui.py to server_configuration_ui.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/build_icons.py to build_icons.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/preferences.py to preferences.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/server_configuration_dialog.py to server_configuration_dialog.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/serverconfig.py to serverconfig.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/config.py to config.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/notifications.py to notifications.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/app_ui.py to app_ui.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/version.py to version.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/buildnotify.py to buildnotify.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/app_notification.py to app_notification.pyc
    byte-compiling /usr/local/lib/python2.7/dist-packages/buildnotifylib/__init__.py to __init__.pyc
    running install_data
    error: can't copy 'buildnotify.desktop': doesn't exist or not a regular file
    
    ----------------------------------------
  Rolling back uninstall of BuildNotify
Command "/usr/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-9KxW3h/buildnotify/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-ypEgr8-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-9KxW3h/buildnotify/
 
```
And indeed there's no such file:
```
$ sudo su
# cd /tmp/pip-build-9KxW3h/buildnotify
# ls -lR | grep desktop
#
```

**Comment by anaynayak** [comment]
Interestingly, i was able to replicate this on a docker machine with a similar setup but unable to replicate it on another docker image now.

@Bu11etmagnet would you be able to check #40 locally ? even on circleci, the first build should have failed (it was meant to be a failing test). 
See https://circleci.com/gh/anaynayak/buildnotify/17

**Comment by anaynayak** [comment]
Actually I managed to test the fix in a Vagrant VM. Will roll out a new release and then investigate ways to have better checks in place.

**Comment by anaynayak** [comment]
Pushed the fix as 1.0.2 . Tracking development pipeline changes with #41

**Comment by Bu11etmagnet** [comment]
1.0.3 did install successfully.
```
$ sudo -H pip install buildnotify --upgrade
[sudo] password: 
Collecting buildnotify
  Downloading BuildNotify-1.0.3.tar.gz
Building wheels for collected packages: buildnotify
  Running setup.py bdist_wheel for buildnotify ... done
  Stored in directory: /root/.cache/pip/wheels/cb/cd/08/361607cf16c74aeca8db9706fc72bde089f67a94fcd7704a93
Successfully built buildnotify
Installing collected packages: buildnotify
  Found existing installation: BuildNotify 1.0.0
    Uninstalling BuildNotify-1.0.0:
      Successfully uninstalled BuildNotify-1.0.0
Successfully installed buildnotify-1.0.3
```
